### PR TITLE
feat(admin): e2e tests for GROUPOBSERVER

### PR DESCRIPTION
**Pull request by Johaney-s**
* e2e tests for groupobserver

### CODE REVIEW

File: apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+context('Actions', () => {
+  const dbGroupName = 'g-o-test-group';
+  const dbGroupMember = 'g-o-testgroupmember';
+  const dbGroupAdmin = 'g-o-testgroupadmin';
+  const dbVoName = 'g-o-test-vo';
+  const dbApplicationItemTextFieldName = 'g-o-input-test';
+  const dbExtsource = 'internal';
+
+  before(() => {
+    if (Cypress.env('BA_USERNAME_GROUP_MANAGER')) {
```
Comment (HejdaJakub) [inline apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js:12]:
```suggestion
    if (Cypress.env('BA_USERNAME_GROUP_OBSERVER')) {
```

File: apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  beforeEach(() => {
+    cy.visit('home').get(`[data-cy=${dbGroupName}]`).click();
+  });
+
+  it('test get member', () => {
+    cy.get('[data-cy=members]')
+      .click()
+      .get(`[data-cy=${dbGroupMember}-firstName-td]`)
+      .should('exist')
+      .click();
```
Comment (HejdaJakub) [inline apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js:29]:
```suggestion
```

File: apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      .get(`[data-cy=${dbApplicationItemTextFieldName}-shortname-td]`)
+      .should('exist');
+  });
+
+  it('test list applications', () => {
+    cy.get('[data-cy=applications]')
+      .click()
+      .get(`[data-cy=${dbGroupName}-groupname-td]`)
+      .should('exist')
+      .click();
```
Comment (HejdaJakub) [inline apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js:77]:
```suggestion
```

**Review by HejdaJakub — CHANGES_REQUESTED** [review]
Just a few minor suggestions.

File: apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js (hunk 2 above)
Comment (Johaney-s) [reply apps/admin-gui-e2e/src/e2e/group-management-group_observer.cy.js]: reused existing member's detail element to check observer makes it to the detail page

**Review by HejdaJakub — APPROVED** [review]

**Pull request merged**